Game.gd

```diff
@@ -175,6 +202,12 @@ func character_attack():
 	yield(current_character, "active_completed")
 	set_game_state(GAME_STATE.UNIT_ATTACK)
 
+func character_wait():
+	current_character.active = true
+	current_character.state = 3
```

Comment:
import character enums into game.gd?

Reply:
https://godotengine.org/qa/40827/how-to-declare-a-global-named-enum
Can only kinda be done right now, so skipping.